assets/runtime/env-defaults

```diff
@@ -5,6 +5,7 @@ DEBUG=${DEBUG:-$DEBUG_ENTRYPOINT}
 ## GITLAB CORE
 GITLAB_TEMP_DIR="${GITLAB_DATA_DIR}/tmp"
 GITLAB_BACKUP_DIR="${GITLAB_BACKUP_DIR:-$GITLAB_DATA_DIR/backups}"
+GITLAB_BACKUP_DIR_CHOWN=true
```

Comment:
This doesn't allow to overwrite it. I think you should do
```bash
GITLAB_BACKUP_DIR_CHOWN=${GITLAB_BACKUP_DIR_CHOWN:-true}
```

Reply:
@solidnerd indeed thanks good spot - pushed the fix